### ISSUE TITLE
Try out Probot and its capabilities

### ISSUE THREAD

**Issue by rodakrafal** [rodakrafal/Zastosowanie-Informatyki-w-Gospodarce#1]

**Comment by rodakrafal** [comment]
@PewPewBartula and I have created a Probot app using `npx create-probot-app my-first-app` and configured it using the `basic-ts` for Typescript support. The process of running the app was rather straightforward, but there were some challenges connected with the Typescript implementation of Probot. Some of those are described in this GitHub [issue](https://github.com/probot/create-probot-app/issues/808) inside of Probot's repository. There are mostly connected with the removal of some APIs. Even though, after overcoming those challenges, we encountered a problem with Probot just not working. The issue turned out to be simple `WEBHOOK_PROXY_URL` that needed to be placed into the console with the corresponding URL.

Also, something to consider is that Probot is using mostly Typescript or JavaScript, and it is not an ideal choice for the team, mostly because most of the team members would have to get acquainted with the language.

**TLDR**: did work, but with some problems. I believe there are better ways to get information about repo, ex. dissections.
